gradle/dependencies.gradle

```diff
@@ -168,6 +169,8 @@ ext {
 
       svgParser                 : "com.caverock:androidsvg-aar:${version.svgParser}",
 
+      glide                     : "com.github.bumptech.glide:glide:${version.glide}",
```

Comment:
What's the process for adding a new dependency? Do we have to discuss it with legal team in terms of license and stuff?

Reply:
I discussed with @kmadsen beforehand and he was fine with using Glide. Maybe @LukasPaczos can add something. 
By the way, Glide is recommended by Google for downloading images here: https://developer.android.com/develop/ui/views/graphics.

Reply:
I'm not against it, I just wonder how it's usually done.

Reply:
Yeah the concern can be with the licence, adding a third party dependency where we need to first purchase a license. For example, mapbox has a [terms of service](https://www.mapbox.com/legal/tos#[CapCap]) in order to use the software (some software is free to use like [mapbox-java](https://github.com/mapbox/mapbox-java/blob/main/LICENSE))

That's why the licence file is being updated with this PR https://github.com/mapbox/mapbox-navigation-android/blob/main/LICENSE.md, we are using [The Apache Software License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0) in a bunch of places already. The "Redistribution" part of that licence is most relevant and it basically says we need to keep a `LICENSE.md` (but i'm not a lawyer so ask a lawyer for more accurate explanation) 

Another reference to the glide licence https://github.com/bumptech/glide/blob/master/LICENSE, but it basically says that you can't sue them if you lose money because it has bugs.

Reply:
@abhishek1508 or @Guardiola31337 may have better explanations here. considering that they added the licence gen tool https://github.com/mapbox/mapbox-navigation-android/pull/2002